patch/kernel/archive/rockchip-rk3588-6.10/0026-RK3588-Add-VPU121-H.264-Decoder-Support.patch

```diff
@@ -289,59 +252,3 @@ index 111111111111..222222222222 100644
 -- 
 Armbian
 
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Sebastian Reichel <sebastian.reichel@collabora.com>
-Date: Thu, 13 Jun 2024 15:48:47 +0200
-Subject: arm64: dts: rockchip: Add VPU121 support for RK3588
-
-From: Jianfeng Liu <liujianfeng1994@gmail.com>
-
-Enable Hantro G1 video decoder in RK3588's devicetree.
-
-Tested with FFmpeg v4l2_request code taken from [1]
-with MPEG2, H.264 and VP8 samples.
-
-[1] https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/multimedia/ffmpeg/patches/v4l2-request/ffmpeg-001-v4l2-request.patch
-
-Signed-off-by: Jianfeng Liu <liujianfeng1994@gmail.com>
-Tested-by: Hugh Cole-Baker <sigmaris@gmail.com>
-Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.com>
----
- arch/arm64/boot/dts/rockchip/rk3588s.dtsi | 21 ++++++++++
- 1 file changed, 21 insertions(+)
-
-diff --git a/arch/arm64/boot/dts/rockchip/rk3588s.dtsi b/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
-index 111111111111..222222222222 100644
---- a/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
-+++ b/arch/arm64/boot/dts/rockchip/rk3588s.dtsi
-@@ -1282,6 +1282,27 @@ power-domain@RK3588_PD_SDMMC {
- 		};
- 	};
- 
-+	vpu121: video-codec@fdb50000 {
-+		compatible = "rockchip,rk3588-vpu121", "rockchip,rk3568-vpu";
```

Comment:
I think we should change compatible instead of removing it completely.

Reply:
I just tested with rk3399-vpu's compatible and it works.